### DISCUSSION TITLE
Is there a new release planned?

### DISCUSSION THREAD

**Discussion by akallabeth** [tristanheaven/gtkhash#146] [Q&A] [answered]
Hi,
the last release is now quite a while back and I´d like to ask if a new release is planned?
(there seem to be enough changes on `master` to justify that)

**Comment by tristanheaven** [comment, marked as answer]
Agree, there are a good number of fixes to release before porting to GTK4 (which will probably introduce new bugs).

I had thought snap translations were broken on master, but that seems to be working now.

Maybe tomorrow.
